projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -118,92 +136,96 @@ export default function MediaSection() {
+					</div>
+				);
+			}
+		},
+		[ mediaData, metaData, onRemoveMedia ]
+	);
 
-	const setMediaRender = useCallback(
-		( { open } ) => (
+	const renderPicker = useCallback(
+		open => (
 			<div className={ styles.container }>
-				<Button className={ ! mediaObject ? styles.toggle : styles.preview } onClick={ open }>
-					{ mediaObject && renderPreview() }
-					{ ! mediaObject && ( attachedMedia.length ? <Spinner /> : ADD_MEDIA_LABEL ) }
-				</Button>
+				{ ! attachedMedia.length ? (
+					<>
+						<Button
+							variant="secondary"
+							size="small"
+							className={ mediaObject && styles.preview }
+							onClick={ open }
+						>
+							{ ! attachedMedia.length && ADD_MEDIA_LABEL }
+						</Button>
+						<span>{ __( 'Add an image or video', 'jetpack' ) }</span>
+					</>
+				) : (
+					<Spinner />
+				) }
 			</div>
 		),
-		[ mediaObject, attachedMedia, renderPreview ]
+		[ mediaObject, attachedMedia ]
 	);
 
-	const replaceMediaRender = useCallback(
-		( { open } ) => (
-			<Button onClick={ open } variant="secondary">
-				{ REPLACE_MEDIA_LABEL }
-			</Button>
-		),
-		[]
+	const setMediaRender = useCallback(
+		( { open } ) => ( mediaObject ? renderPreview( open ) : renderPicker( open ) ),
+		[ mediaObject, renderPreview, renderPicker ]
 	);
 
 	const onDismissClick = useCallback( () => setValidationError( null ), [] );
 
 	return (
-		<div className={ styles.wrapper }>
-			<MediaUploadCheck>
-				<MediaUpload
-					title={ ADD_MEDIA_LABEL }
-					onSelect={ onUpdateMedia }
-					allowedTypes={ allowedMediaTypes }
-					render={ setMediaRender }
-					value={ attachedMedia[ 0 ]?.id }
-				/>
-				{ mediaObject && (
-					<>
-						<MediaUpload
-							title={ REPLACE_MEDIA_LABEL }
-							onSelect={ onUpdateMedia }
-							allowedTypes={ allowedMediaTypes }
-							render={ replaceMediaRender }
-						/>
-						<Button onClick={ onRemoveMedia } variant="link" isDestructive>
-							{ REMOVE_MEDIA_LABEL }
-						</Button>
-					</>
-				) }
-				{ validationError && (
-					<Notice
-						className={ styles.notice }
-						isDismissible={ true }
-						onDismiss={ onDismissClick }
-						status="warning"
-					>
-						<p>{ validationErrorMessages[ validationError ] }</p>
-					</Notice>
-				) }
-				{ ! mediaObject && (
-					<Fragment>
-						<Text variant="title-small">{ __( 'Max image size', 'jetpack' ) }</Text>
-						<Notice className={ styles.max_notice } isDismissible={ false } status="info">
-							<Text>{ ` ${ maxImageSize } Mb` }</Text>
-						</Notice>
-						<Text variant="title-small">{ __( 'Allowed types', 'jetpack' ) }</Text>
-						<Notice className={ styles.max_notice } isDismissible={ false } status="info">
-							<Text>{ ` ${ allowedMediaTypes.join( ', ' ) }` }</Text>
+		<ThemeProvider>
+			<BaseControl label={ __( 'Media', 'jetpack' ) } className={ styles.wrapper }>
+				<MediaUploadCheck>
+					<p className={ styles.subtitle }>
+						{ __( 'Choose a visual to accompany your post.', 'jetpack' ) }
```

Comment:
Totally possible I am looking at an old iteration of the design, but I don't see the `Choose a visual to accompany your post.` in the design, do you think we should remove it?

Reply:
It's missing from p1HpG7-jqQ-p2 but it was added in the wrap-up (see here: NIq2WbeH5khnwLVaHbUvSX-fi-3150%3A6735)